MeshGeoToolsLib/GeoMapper.cpp

```diff
@@ -185,27 +228,20 @@ double GeoMapper::getMeshElevation(
 	return (*(_surface_mesh->getNode(pnt->getID())))[2];
 }
 
-unsigned getIndexInPntVec(GeoLib::Point const* const pnt,
-                          std::vector<GeoLib::Point*> const& points)
-{
-	auto it (std::find(points.begin(), points.end(), pnt));
-	return static_cast<unsigned>(std::distance(points.begin(), it));
-}
-
 std::unique_ptr<std::vector<GeoLib::Polyline*>> copyPolylinesVector(
-    std::vector<GeoLib::Polyline*> const& polylines,
-    std::vector<GeoLib::Point*> const& points)
+	const std::vector<GeoLib::Polyline*>* polylines,
```

Comment:
Actually I liked the old signature better. But there probably is a reason for switching to pointers.